tests/e2e/scenarios/test_reborn_webui_v2_legacy_extensions.py

```
@@ -445,17 +445,19 @@ async def _open_card_menu(card):
     return card.get_by_role("menu")
 
 
-async def _capture_next_confirm(page, *, accept: bool):
-    loop = asyncio.get_running_loop()
-    dialog_future = loop.create_future()
+async def _resolve_remove_confirm(page, *, accept: bool, name: str):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_resolve_remove_confirm`

Add return type annotation: `None`

(ANN202)